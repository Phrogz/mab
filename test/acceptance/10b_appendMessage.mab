p( "Test#10b" )
setSlot( "myExpr", Expression new )
setSlot( "myMessage", Message new( "p", "Hello, world!" ) )
myExpr appendMessage( myMessage )
myExpr eval