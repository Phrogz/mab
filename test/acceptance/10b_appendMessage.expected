Test#10b
Hello, world!